src/relay/relay.md

```diff
@@ -0,0 +1 @@
+TODO
```

Comment:
Should be 2 chapters at least, one for a client and one for a server.

TODO: copy from https://toktok.ltd/spec#tcp-server

Reply:
Going to be subchapters (just like onion).